src/impi/core.cljs

```diff
@@ -227,6 +260,25 @@
        (map #(build! index attr %))
        (replace-children container)))
 
+; Does not cache individual shapes as any change requires a full redraw
+(defmethod update-prop! :pixi.graphics/shapes 
+  [graphics-obj _ _ shapes]
+  (.clear graphics-obj)
+  (doseq [{fill       :pixi.shape/fill
```

Comment:
Awesome. I didn't know that.

 I usually prefer restructuring inline, but I'd be willing to change it if that fits your style better.